packages/itwinui-css/src/tile/tile.scss

```diff
@@ -106,17 +109,10 @@
   }
 
   //#region Statuses
-  &.iui-progress {
-    cursor: progress;
-    font-style: italic;
-  }
-
+  &.iui-loading,
   &[aria-disabled='true'] {
-    cursor: not-allowed;
-
-    .iui-tile-name-label {
-      color: var(--iui-color-foreground-4);
-    }
+    --_iui-tile-title-text-color: var(--iui-color-foreground-4);
+    --_iui-tile-body-text-color: var(--iui-color-foreground-4);
```

Comment:
I'm not too sure of css variable scopes, but is --_iui-tile-body-text-color already defined for this selector scope on [line 9](https://github.com/iTwin/iTwinUI/pull/765/files#diff-875f9cda040043eba8e927b119436e84c98074bbd862870ed84930dd5265dc25R9)? Because we are still in the code block of [line 9](https://github.com/iTwin/iTwinUI/pull/765/files#diff-875f9cda040043eba8e927b119436e84c98074bbd862870ed84930dd5265dc25R7)

Reply:
Ah, good catch, I have removed the redundant line 115.